### PR TITLE
Rebalances a vast oversight with one of the game's weapons

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Makes sure the game is fairly balanced again.
## Why It's Good For The Game
Honestly I'm surprised this weapon went unchanged for so long with no downsides.
## Changelog
:cl: Wallem
balance: Fixes a glaring oversight with the entrenching tool's balancing.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
alright now add the kritzkrieg

File: code/modules/mining/equipment/mining_tools.dm [hunk 1]
```diff
@@ -204,6 +204,7 @@
 	. = ..()
 	. += span_notice("Use in hand to switch configuration.")
 	. += span_notice("It functions as a [tool_behaviour] tool.")
+	. += span_notice("<i>This weapon has no random critical hits.</i>")
```
Comment (carlarctg) [inline code/modules/mining/equipment/mining_tools.dm:207]:
```suggestion
	. += span_danger("<i>This weapon has no random critical hits.</i>")
```

or whatever is red in this codebase

Comment (san7890) [reply code/modules/mining/equipment/mining_tools.dm:207]: yeah that's red enough

**Comment by CRITAWAKETS** [comment]
HOW DID THEY LET THAT IN!????

**Comment by VladinXXV** [comment]
add rocket jumping and crits on rocket jump NOW. market garden ops NOW.

**Pull request merged**